app/templates/components/forms/admin/settings/system-form.hbs

```diff
@@ -33,10 +33,9 @@
     {{input type='text' name='tag_line' value=settings.tagline}}
   </div>
   <div class="ui hidden divider"></div>
-
   {{forms/admin/settings/system/social-media-token settings=settings}}
-
-
+  <div class="ui hidden divider"></div>
+  {{forms/admin/settings/system/captcha-option settings=settings}}
```

Comment:
you didn't change the name here, it won't work, nothing will get rendered.

Reply:
@CosmicCoder96 Left due to ignorance, Did the change please check